bug_report_page.php

```diff
@@ -230,16 +230,21 @@
 layout_page_begin( __FILE__ );
 
 $t_form_encoding = '';
+$t_form_class = '';
 if( $t_show_attachments ) {
 	$t_form_encoding = 'enctype="multipart/form-data"';
+	$t_form_class = 'class="dropzone-form"';
 }
 ?>
 <div class="col-md-12 col-xs-12">
 <form id="report_bug_form"
 	method="post" <?php echo $t_form_encoding; ?>
 	action="bug_report.php?posted=1"
-	class="dropzone-form"
-	<?php print_dropzone_form_data() ?>>
+	<?php echo $t_form_class ?>
+	<?php if( $t_show_attachments ) {
```

Comment:
Why two php blocks next to each other?  Can they be combined?  Thought that may change with the previous suggestions.